File: src/main/java/space/ryzhenkov/colorfulanvils/ColorfulAnvils.java
package space.ryzhenkov.colorfulanvils;

import net.fabricmc.api.ModInitializer;
import org.slf4j.LoggerFactory;

public class ColorfulAnvils implements ModInitializer {
    public void onInitialize() {
        LoggerFactory.getLogger("ColorfulAnvils").info("Time to bring color juice for anvils!");
    }
}

File: src/main/kotlin/space/ryzhenkov/colorfulanvils/ColorfulAnvils.kt
package space.ryzhenkov.colorfulanvils

import net.fabricmc.api.ModInitializer
import java.util.logging.Logger

class ColorfulAnvils:ModInitializer {
    override fun onInitialize() {
        Logger.getLogger("ColorfulAnvils").info("Time to bring color juice for anvils!")
    }
}

